Sources/HaCWebsiteLib/Views/LandingPage/LandingFeatures/ContinuousIntegrationWorkshop.swift

```diff
@@ -0,0 +1,17 @@
+import HaCTML
+import Foundation
+
+extension LandingFeatures {
+  static var continuousIntegrationWorkshop: LandingFeature {
+    return EventFeature(
+      eventPeriod: DateInterval(
+        start: Date.from(year: 2018, month: 2, day: 5, hour: 13, minute: 15    ),
+        duration: 1.5 * 60 * 60
+      ),
+      eventLink: "https://www.facebook.com/events/181764052556196/",
+      liveLink: "/workshops/continuous-integration",
+      hero: WorkshopManager.workshops["workshop-continuous-integration"]!.hero,
```

Comment:
Not safe to force unwrap here since it relies on GitHub being up and on the continuous integration workshop data being in a valid format. I'd maybe make LandingFeatures.features be an array of optionals and then have current feature find the first non-optional one with a future expiry date.